src/pymoca/backends/casadi/model.py

```diff
@@ -11,7 +11,8 @@
 
 logger = logging.getLogger("pymoca")
 
-CASADI_COMPARISON_DEPTH = 100
+CASADI_COMPARISON_DEPTH = 1000000
```

Comment:
use `sys.maxsize`?

Reply:
Well, I think even 1 million is a bit large. Maybe 1000 or 10k is high enough. What I want to avoid is infinite loops, and sys.maxsize will be a very long wait leading to users blaming pymoca for being slow. Instead, we should just raise an Exception indicating that something is wrong with the model.

Note that the reason for these deep graphs (with RTC-CF models) is CasADi's implementation of `linspace`. I have a pending PR on their side addressing this: https://github.com/casadi/casadi/pull/2338

Reply:
I always shudder when I see arbitrary constants ...